index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portifolio</title>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+    <header class="toolbar">
+        <a href="index.html" class="name button"> SOLOMON</a>
+        <nav>
+          <img class="huberger" src="/assets/Union.png" alt="Dropdown menu">
+        </nav>
+    </header>    
+     <section class="first-page">
+           <div class="headline">
+             <h1 class="hey">Hey There  I'm Solomon</h1>
+             <h2 class="softwaredev"> I am software developer</h2>
+             <p class="headline-p">I can help you build a product , feature or website Look through some of my work
+                 and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+             </p>
+          </div>
+          <ul class="socila-link">
+            <li class="social"><a href="https://github.com/solog0039"><img  src="./assets/socialmedia/github.png" alt="Github"></a></li>
+            <li class="social"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img  src="./assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
+            <li class="social"><a href="##"><img  src="./assets/socialmedia/tump.png" alt="tum"></a></li>
+            <li class="social"><a href="https://twitter.com/Solomon57320119"><img  src="./assets/socialmedia/tiwtter.png" alt="tiwtter"></a></li>
+            <li class="social"><a href="##"><img  src="./assets/socialmedia/m.png" alt="M"></a></li>
+          </ul>
+</section>
+<section class="cards">
```

Comment:
- [ ] I notice you're using the `background-image` instead of `img` tag for some of the project sections, since it's part of the project content it's so vital to use the `img` tag, the background image is used for design and aesthetics purposes only, please refactor each project item to contain an `img` tag instead 💪.